dtmi/seeedkk/wioterminal/wioterminal_smellsensor-1.json

```diff
@@ -0,0 +1,54 @@
+{
+    "@id": "dtmi:seeedkk:wioterminal:wioterminal_smellsensor;1",   
+    "@type": "Interface",
+    "displayName": "Seeed Wio Terminal",
```

Comment:
nit: Update the `displayName` to avoid duplicates with other interfaces in this folder